.adr/0010-interest-bearing-channel.md

```diff
@@ -0,0 +1,128 @@
+
+Fees on liquidity are required because:
+
+- a given intermediary channel has hard limits on the number of pending transactions it can process at a time (~400)
+- a given intermediary channel has only so much capacity to adjust its balance in one direction or the other (IE, to forward payments)
+
+A channel in a PCN is capacity limited according to its current balance. The limitation is asymmetric: a one BTC channel whose balance is 0.5-0.5 can forward payments up to 0.5 BTC in either direction, but the same channel at a different time with a balance of 0.1-0.9 can forward only 0.1 to the right (and 0.9 to the left). Most LN channels _expect_ to be bi-directional in practice, and for the general flow of payments back and forth to keep the channel balanced on average.
+
+Two potential inefficiencies in this fee structure come to mind:
+
+First, the existing LN fee structure does not specifically account for unbalanced channels. In the case of the 0.1-0.9 channel, the channel operators should much prefer forwarding payments to the left, since those payments re-balance the channel. That preference _could_ be expressed in the market as a discounted fee for leftward payments and a markup for rightward payments, but it is not.
+
+Second, the scalar multiplier for payments does not account for the duration of the capital lockup, which is an inaccuracy. In a PCN, this price inaccuracy is generally small, and always bounded: lockups are reasonably short in the optimistic case (where payment is successfully routed) and come with a specified timeout for the failure case. This is _very different_ from a SCN, where the default case is for lockups with non-trivial duration (channel run-time). In the SCN case, a scalar per-lockup fee mechanism is generally inaccurate, and the inaccuracy is unpredictable.
+
+# Decision Criteria
+
+There are a number of decision dimensions here:
+
+1. purpose-fit of introduced economic model (does it drive pro-network behaviors?)
+2. security implications
+3. efficiency / gas consumption implications
+4. programmability / tunability of implementation
+5. ergonomics for
+   - channel application developers
+   - SCN wallet developers
+   - network participants
+6. software lock-in, protocol flexibility, composability
+
+... plus the usual meta-considerations such as engineering effort.
+
+# Considered Options
+
+## Fee Buckets as a Protocol Primitive
+
+> (See related [Notion Doc](https://www.notion.so/statechannels/RFC-20-Introduce-a-fee-model-for-intermediaries-by-modifying-reclaim-dce28ac74d764fcf93db7d6b5cf05b3a))
+>
+> The linked document describes a mechanism where a channel _payer_ (Alice) fronts a specified (potential run-time negotiated) fee for each funded virtual channel, which is distributed among route intermediaries at channel closing time. This is enforced at the protocol level via a modification to `reclaim`.
+
+Against the above decision criteria: (on 😀 🙂 😐 🙁 😢
```

Comment:
❤️ 

Nit: could we do a table or hack the spacing so that the symbols line up in a nice way?